src/applications/mhv-secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -239,13 +240,18 @@ const ComposeForm = props => {
           sendData = JSON.stringify(messageData);
         }
         dispatch(sendMessage(sendData, attachments.length > 0))
-          .then(() =>
-            navigateToFolderByFolderId(
-              currentFolder?.folderId || DefaultFolders.INBOX.id,
-              history,
-            ),
-          )
-          .catch(setSendMessageFlag(false));
+          .then(() => {
+            setTimeout(() => {
+              navigateToFolderByFolderId(
+                currentFolder?.folderId || DefaultFolders.INBOX.id,
+                history,
+              );
+            }, 1000);
```

Comment:
if this `setTimeout` here is intentional, let's just leave a comment for the future reference describing why we need it

Reply:
I have also noticed a weird behavior with this `setTimeout` delay where it conflicts with a timeline of the success banner. Let's say a message send request completed before this `1s` delay - we start seeing success banner while the loading spinner is still there. Not sure if this may be an a11y problem

Reply:
actually, that is also the case if the request is longer than 1s

Reply:
this was a cause to update the tests.

Reply:
I guess my questions was more re: the technical implementation aspect to @KolbyVA

Reply:
exactly, just wanted to show the reason of my involving to this PR